guide/src/whirlwind-tour/basic-usage.md

```diff
@@ -170,6 +164,18 @@ $ npm run serve
 If you open [http://localhost:8080](http://localhost:8080) in a browser you should see a `Hello, world!`
 dialog pop up!
 
+Notice that `html-webpack-plugin` has generated an HTML page which includes `index.js`.
+
```

Comment:
I think I would be a little sad to see the index.html file removed.  I liked it because (1) it showed that there wasn't anything special in it and (2) I immediately knew where to start expanding my application.

So maybe you could add a line or two of instruction text that says something like "If you want to use your own index.html file instead of the generated one, do steps XYZ"

Reply:
Done